pom.xml

```diff
@@ -873,13 +873,6 @@
 
         <pluginManagement>
             <plugins>
-                <plugin>
-                    <groupId>org.apache.rat</groupId>
-                    <artifactId>apache-rat-plugin</artifactId>
-                    <configuration>
-                        <skip>true</skip>
-                    </configuration>
-                </plugin>
```

Comment:
I checked that we just need to this code, and nothing need to change in `license-cehck.yml`, then everything works as expected https://github.com/alibaba/fluss/commits/ci-rat-check/. It seems I added this code by accident in https://github.com/alibaba/fluss/commit/bb7fbc2a532d25ff0e838e0976b1266130288367. 

Could you update this PR again?